### PR TITLE
Update pillow docker

### PR DESCRIPTION
**Pull request by hkctkuy**

### CODE REVIEW

File: projects/pillow/pillow.toml [hunk 1]
```diff
@@ -14,8 +14,10 @@
...
 
-exit-on-time=3600
+exit-on-time = 3600
 
 [atheris]
 path = "/Pillow/Tests/oss-fuzz/fuzz_pillow.py"
-args = "/out/fuzz_pillow_seed_corpus/ -dict=/out/fuzz_pillow.dict"
+args = "/out/fuzz_pillow_seed_corpus/ -dict=/out/fuzz_pillow.dict -jobs=1000 -workers=4"
+
+cmin=false
```
Comment (SweetVishnya) [inline projects/pillow/pillow.toml:23]: why cmin is false?

**Review by SweetVishnya — CHANGES_REQUESTED** [review]

File: projects/pillow/build.sh [hunk 2]
```diff
@@ -24,4 +24,27 @@ export CXX="clang++"
...
+      --add-binary /usr/local/lib/libwebp.so.7:. \
+      --add-binary /usr/local/lib/libwebpdemux.so.2:. \
+      --add-binary /usr/local/lib/libwebpmux.so.3:. \
+      --add-binary /usr/local/lib/libxcb.so.1:. \
+      --distpath $OUT --onefile --name $fuzzer_package $fuzzer
+done
+
+# Collect corpus
+find Tests/images Tests/icc -print | zip -q $OUT/fuzz_pillow_seed_corpus.zip -@
+find Tests/fonts -print | zip -q $OUT/fuzz_font_seed_corpus.zip -@
```
Comment (SweetVishnya) [inline projects/pillow/build.sh:50]: do not zip corpus, just copy it to directory

File: projects/pillow/pillow.toml (hunk 1 above)
Comment (hkctkuy) [reply projects/pillow/pillow.toml]: Fixed

File: projects/pillow/build.sh (hunk 2 above)
Comment (hkctkuy) [reply projects/pillow/build.sh]: Done

**Review by SweetVishnya — APPROVED** [review]

**Pull request merged**